packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.types.ts

```diff
@@ -24,9 +24,10 @@ export interface IVerticalBarChartProps extends ICartesianChartProps {
   onRenderCalloutPerDataPoint?: IRenderFunction<IVerticalBarChartDataPoint>;
 
   /**
-   * Width of each bar in the chart.
+   * Width of each bar in the chart. When set to `auto`, the bar width is calculated from padding values.
+   * @default 16
    */
-  barWidth?: number;
+  barWidth?: number | 'default' | 'auto';
```

Comment:
we are not checking for the 'default' literal anywhere explicitly. Why so

Reply:
The getBarWidth utility returns 16 by default when the barWidth prop is not a number. The 'auto' case is handled separately, but the 'default' case is treated like any other fixed bar width scenario after conversion to a number by the utility.

Reply:
add this additional check that if barWidth === 'default' then use 16px. To ensure future issues like 
if (this._xAxisType !== XAxisTypes.NumericAxis)  
dont occur